<launch>
    <arg name="sim" value="true"/>
    <arg name="gui" default="true"/>

    <arg name="start_home" default="true"/>
    <arg name="scene" default="false"/>
    <arg name="initial_z" default="0.7751"/>
    <arg name="world" if="$(arg scene)"/>
    <arg name="world" value="$(find sr_description_common)/worlds/shadowhand.world" unless="$(arg scene)"/>

    <env name="GAZEBO_MODEL_PATH" value="$(find sr_description_common)/models/" />
    <env name="GAZEBO_MODEL_DATABASE_URI" value="" />
    <env name="MESH_WORKSPACE_PATH" value="$(find sr_description_common)/models/" />

    <include  file="$(find sr_robot_launch)/launch/sr_ur_arm_box.launch">
        <arg name="gui" value="$(arg gui)"/>
        <arg name="sim" value="$(arg sim)"/>
        <arg name="initial_z" value="$(arg initial_z)"/>
        <arg name="scene" value="$(arg scene)"/>
        <arg name="world" value="$(arg world)"/>
        <arg name="start_home" value="$(arg start_home)"/>
    </include>
</launch>